applications/nrf_desktop/doc/info.rst

```diff
@@ -0,0 +1,42 @@
+.. _info:
+
+Info module
+###########
+
+Use the Info module to:
+
+* handle the role of the final subscriber for the :ref:`config_channel` events,
+* provide the board name through the :ref:`config_channel`.
+
+Module Events
+*************
+
++------------------+--------------------------------+---------------+------------------------+------------------+
+| Source Module    | Input Event                    | This Module   | Output Event           | Sink Module      |
++==================+================================+===============+========================+==================+
+| :ref:`hids`      | ``config_event``               |  ``info``     |                        |                  |
++------------------+                                |               |                        |                  |
+| :ref:`usb_state` |                                |               |                        |                  |
++------------------+--------------------------------+               |                        |                  |
+| :ref:`hids`      | ``config_fetch_request_event`` |               |                        |                  |
++------------------+                                |               |                        |                  |
+| :ref:`usb_state` |                                |               |                        |                  |
++------------------+--------------------------------+               +------------------------+------------------+
+|                  |                                |               | ``config_fetch_event`` | :ref:`hids`      |
+|                  |                                |               |                        +------------------+
+|                  |                                |               |                        | :ref:`usb_state` |
++------------------+--------------------------------+---------------+------------------------+------------------+
+
+Configuration
+*************
+
+The Info module is required by the :ref:`config_channel`.
+The module is enabled with the same Kconfig option as the :ref:`config_channel` (``CONFIG_DESKTOP_CONFIG_CHANNEL_ENABLE``).
+
+Implementation details
+**********************
+
+The Info module must be the only final subscriber for the ``config_event`` and ``config_fetch_request_event``.
```

Comment:
event manager supports only one final subscriber for an event so adding `only` is not correct

Reply:
The only may be misleading here, removed.